lua/telescope-alternate/presets.lua

```diff
@@ -1,34 +1,51 @@
 local M = {}
 
 M.rails = {
-  { "app/models/(.*).rb", {
-    { "app/controllers/**/*[1:pluralize]_controller.rb", "Controller" },
-    { "app/views/[1:pluralize]/*.html.erb", "View" },
-    { "app/helpers/[1]_helper.rb", "Helper" },
-    { "app/serializers/**/*[1]_serializer.rb", "Serializer" }
+  { 'app/models/(.*).rb', {
+    { 'app/controllers/**/*[1:pluralize]_controller.rb', 'Controller' },
+    { 'app/views/[1:pluralize]/*.html.erb', 'View' },
+    { 'app/helpers/[1]_helper.rb', 'Helper' },
+    { 'app/serializers/**/*[1]_serializer.rb', 'Serializer' },
+    { 'spec/requests/**/*[1:pluralize]_spec.rb', 'Request Test' },
+    { 'spec/models/**/*[1:singularize]_spec.rb', 'Model Test' },
+    { 'spec/factories/**/*[1:pluralize].rb', 'Factory' }
   } },
-  { "app/controllers(.*)/(.*)_controller.rb", {
-    { "app/models/**/*[2:singularize].rb", "Model" },
-    { "app/views/[1][2]/*.html.erb", "View" },
-    { "app/helpers/**/*[2]_helper.rb", "Helper" },
-    { "app/serializers/**/*[2]_serializer.rb", "Serializer" }
+  { 'app/controllers(.*)/(.*)_controller.rb', {
+    { 'app/models/**/*[2:singularize].rb', 'Model' },
+    { 'app/views/[1][2]/*.html.erb', 'View' },
+    { 'app/helpers/**/*[2]_helper.rb', 'Helper' },
+    { 'app/serializers/**/*[2]_serializer.rb', 'Serializer' },
+    { 'spec/requests[1]/[2]*_spec.rb', 'Request Test' },
```

Comment:
Leave all spec relates to the rspec preset.  So who uses minitest can do differently